src/formatDate.js

```diff
@@ -50,7 +50,37 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+  const oldDateSorted = {};
+  let newDate = '';
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat[i] === 'YY' && toFormat.includes('YYYY')) {
```

Comment:
It would be cool to put the "magic data" in constants, for a better understanding of the code.